editor/src/scene_viewer/mod.rs

```diff
@@ -827,14 +826,21 @@ impl SceneViewer {
                                     }
                                 }
                             }
-                            WidgetMessage::MouseMove { pos, .. } => {
+                            WidgetMessage::MouseMove { pos, state, .. } => {
                                 let rel_pos = pos
                                     - engine
                                         .user_interfaces
                                         .first()
                                         .node(self.scene_gizmo_image)
                                         .screen_position();
-                                self.scene_gizmo.on_mouse_move(rel_pos, engine);
+                                self.scene_gizmo.on_mouse_move(rel_pos, engine, None);
+                                if state.left == ButtonState::Pressed {
+                                    self.scene_gizmo.on_mouse_move(
```

Comment:
Instead of double call just change signature of `on_mouse_move` to accept mouse button and its state and check for pressed/unpressed in `on_mouse_move` itself.